src/cli/utils/flags.go

```diff
@@ -167,6 +177,36 @@ func AddSiteFlag(cmd *cobra.Command) {
 		"Backup data by site URL; accepts '"+Wildcard+"' to select all sites.")
 }
 
+func AddMaintenanceSafetyFlag(cmd *cobra.Command) {
+	fs := cmd.Flags()
+	fs.StringVar(
+		&MaintenanceSafetyFV,
+		MaintenanceSafetyFN,
+		string(control.FullSafety),
+		"Disable safety measures to run maintenance faster. Improper usage can break a repository")
```

Comment:
admittedly, there's a part of me that wants it to say `but can bork the repository`
```suggestion
		"Set the level of safety measures; lower safety makes maintenance faster, but can break the repository")
```

Reply:
We should look at some of the Kopia language here and make this more specific. For Kopia, safety means that you need to make sure no concurrent operations are happening 

https://kopia.io/docs/advanced/maintenance/

Reply:
@ashmrtn - let's revisit this. We have safety options in the SDK but let's not expose in CLI yet.